### DISCUSSION TITLE
How .sdc is related to synthesis?

### DISCUSSION THREAD

**Discussion by tristonw109** [The-OpenROAD-Project/OpenROAD-flow-scripts#788] [Q&A] [answered]
This may be a "stupid" question to ask, however, My professor and I are doing research that involves the OpenROAD flow. I have little experience with general ASIC/IC RTL-to-GDS flows. How does the synopsys design constraints file (.sdc) relate to synthesis? How does the .sdc file affect the flow in general?

**Comment by maliberty** [comment, marked as answer]
Synthesis will get information about the clock and i/o timings.  Its primarily used during techmapping and optimization which is done by abc from within yosys.

**Comment by maliberty** [comment]
In the rest of the flow in OpenRoad it is used to drive clock tree synthesis and timing optimization throughout the flow.
